packages/library-legacy/src/connection.ts

```diff
@@ -989,6 +1011,7 @@ const SimulatedTransactionResponseStruct = jsonRpcResultAndContext(
         }),
       ),
     ),
+    innerInstructions: optional(nullable(array(ParsedInnerInstructionStruct))),
```

Comment:
Does this need to be optional + nullable? Will just optional suffice? Or will it be array, undefined or null in different situations?